skins/cat17/src/sass/common/_variables.scss

```diff
@@ -20,6 +20,8 @@ $brand-footer: #cfdbe2;
 $gray-light: #ddd;
 $brand-error: #812923;
 $brand-error-rgba: rgba(129, 41, 35, 0.2);
+$brand-warning: #812923;
+$brand-warning-rgba: rgba(129, 41, 35, 0.2);
```

Comment:
Seems to be unused.

Reply:
Do you mean just the rgba one? I though that for whatever reason both these are needed. Not?

Reply:
Yes, I just refered to the -rgba one. There is no magic involved, you just have to keep `skins/cat17/node_modules/bootstrap-sass/assets/stylesheets/bootstrap/_variables.scss` in mind. You got lucky, used the "correct" (pre-existing) variable `brand-warning`.

Reply:
Killed the line